tests/pql/test_declarations.py

```diff
@@ -1,9 +1,181 @@
+import pytest
+
 from ats.pql.pql import evaluate_query
 
 
 def test_simple_stmt_declaration():
-    evaluate_query(
+    result = evaluate_query(
         """ stmt s1;
             Select s1 such that Parent(s1, "x")
         """
     )
+
+    assert result[0]["variables"]["s1"] == "stmt"
+
+
+def test_complex_stmt_declaration():
+    result = evaluate_query(
+        """ stmt s1; stmt s2; stmt s3;
+            Select s1 such that Parent(s1, "x")
+        """
+    )
+
+    assert (
+        result[0]["variables"]["s1"] == "stmt"
+        and result[0]["variables"]["s2"] == "stmt"
+        and result[0]["variables"]["s3"] == "stmt"
+    )
```

Comment:
rozbij to na oddzielne asserty. Chcemy wiedziec co konkretnie failuje jak failuje